Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/CheckVersionIntegrityCommand.cs

```diff
@@ -93,18 +93,22 @@ private FileIntegrity CheckFile(AppContentSummaryFile file)
                 return new FileIntegrity(file.Path, FileIntegrityStatus.InvalidVersion);
             }
 
-            string hash = HashCalculator.ComputeFileHash(localPath);
-
-            if (_checkType == IntegrityLevel.HashChecksum && hash != file.Hash)
+            if (_checkType <= IntegrityLevel.HashChecksum)
```

Comment:
For checking whether flag is enabled you should use following code `(_checkType & IntegrityLevel.HashChecksum) == IntegrityLevel.HashChecksum` or `(_checkType & IntegrityLevel.HashChecksum) != 0`.

Which one to use is up to you, IMHO first one is more readable (but takes more space).